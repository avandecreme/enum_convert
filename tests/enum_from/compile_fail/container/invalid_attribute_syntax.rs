use enum_convert::EnumFrom;

enum Source1 {
    Unit,
}

enum Source2 {
    Unit,
}

#[derive(EnumFrom)]
#[enum_from(Source1; Source2)] // Semi colon instead of comma
enum Target {
    Unit,
}

fn main() {}
