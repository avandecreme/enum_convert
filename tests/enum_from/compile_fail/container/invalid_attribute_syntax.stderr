error: expected `,`
  --> tests/enum_from/compile_fail/container/invalid_attribute_syntax.rs:12:20
   |
12 | #[enum_from(Source1; Source2)] // Semi colon instead of comma
   |                    ^
